examples.rkt

```diff
@@ -17,72 +17,73 @@
 ;; Examples
 (define-relation (appendo ab c abc)
   (conde
-    ((== '() ab) (== c abc))
-    ((fresh (a b bc)
-       (== `(,a . ,b) ab)
-       (== `(,a . ,bc) abc)
-       (appendo b c bc)))))
+   ((== '() ab) (== c abc))
+   ((fresh (a b bc)
+           (== `(,a . ,b) ab)
+           (== `(,a . ,bc) abc)
+           (appendo b c bc)))))
```

Comment:
`fresh` is a variable-binding form (similar to `let` or `lambda`), so its body should not be indented so much.  Can you adjust your editor's config to have it use the original indentation?